x-pack/plugins/security_solution/server/search_strategy/security_solution/index.ts

```diff
@@ -28,9 +32,17 @@ export const securitySolutionSearchStrategyProvider = <T extends FactoryQueryTyp
       const queryFactory: SecuritySolutionFactory<T> =
         securitySolutionFactory[request.factoryQueryType];
       const dsl = queryFactory.buildDsl(request);
-      return es
-        .search({ ...request, params: dsl }, options, deps)
-        .pipe(mergeMap((esSearchRes) => queryFactory.parse(request, esSearchRes)));
+      return es.search({ ...request, params: dsl }, options, deps).pipe(
```

Comment:
We now shim totals in the responses in the route handler.
This means that internally, all strategies should work with the new format `{value: 123, relation: 'eq'}`.
We shimmed the types here to avoid test failures.